### ISSUE TITLE
Users and permissions gone after update

### ISSUE THREAD

**Issue by departet** [go-gitea/gitea#14743]
- Gitea version (or commit ref): 1.14.0+dev-714-g7ab6c77b4
- Git version:
- Operating system: Ubuntu 20.04 with Docker
- Database:
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No

Hi,
im running gitea in a docker environment with watchtower. I assume this is where the bug started. I can still log in to gitea and see my reps but i don't have permissions on my organization neither can i create a git in that organization. I don't have admin rights either. When listing the users in the cli i get every user. After manually adding a new admin with the cli and logging into the web i don't see any user besides the freshy created one under "administration - users".
So basically i still have all my data but users are not shown on the web and i don't have admin rights anymore.

I just installed sqlite3 and went into the db. All users (old and new) are present in the db.

![image](https://user-images.githubusercontent.com/68166825/108484960-b27e0500-729c-11eb-87f2-b0827c7ba240.png)
Aside from the user and organization data this is correct.

**Comment by Thoxy67** [comment]
I have the same to me

**Comment by techknowlogick** [comment]
Related to this: https://github.com/go-gitea/gitea/pull/14521 there is a doctor command to resolve this. There is an issue with more details that I can't find right now (about to sleep), but I'm sure someone can link it here. Closing this issue as a dupe of that other issue.
